public/app/features/trails/MetricSelectScene.tsx

```diff
@@ -298,11 +298,11 @@ export class MetricSelectScene extends SceneObjectBase<MetricSelectSceneState> {
       <div className={styles.container}>
         {showHeading && (
```

Comment:
If we get rid of this "heading" and "show heading" we could consider using the `<Field>` wrapper component as before to simplify things.

This is assuming we want this to appear the same when it's at the main "Select Metric" mode and when it's in the "Related Metrics" tab.